src/assets/scripts/client/ui/TutorialView.js

```diff
@@ -258,7 +258,7 @@ export default class TutorialView {
             title: 'Moving Around',
             text: ['To move the middle of the radar screen, use the right click button and drag.',
                 'Zoom in and out by scrolling, and press the middle mouse button or scroll wheel to reset the zoom.',
-                'To select an aircraft when it is in flight, simply left-click.'
+                'To select an aircraft when it is in your control, simply left-click.'
             ].join(' '),
             position: tutorial_position
         });
```

Comment:
The flight strips bay will be closed, the steps describing flight strips should come here

Reply:
Sorry, I don't quite understand?

Reply:
When you start the sim, the flight strip bay on the right will be closed. There's a section later on which explains how to open it, but it should be explained before this step, since it makes reference to the flight strips